### PR TITLE
Added Danish language support and translation

### PR DESCRIPTION
**Pull request by ChristianLieven**
This should get us started!

In the future, one could consider adding more word lists. Especially that [pwned password](https://github.com/n0kovo/danish-wordlists/blob/main/dk_breach.txt) list looked relevant.

### CODE REVIEW

File: packages/languages/da-dk/CHANGELOG.md
Comment (MrWook) [inline packages/languages/da-dk/CHANGELOG.md:1]: This can be cleared

File: packages/languages/de/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { ZxcvbnFactory } from '@zxcvbn-ts/core'
 import * as zxcvbnCommonPackage from '@zxcvbn-ts/language-common'
 import * as zxcvbnDePackage from '@zxcvbn-ts/language-de'
 
-const password = 'somePassword'
+const password = 'nogleAdgangskode'
```
Comment (MrWook) [inline packages/languages/de/README.md:22]: Wrong language pack

**Review by MrWook — COMMENTED** [review]
Thanks for the new language package!

Once this is merged i can add the danish wikipedia dictionary if you don't want to generate it yourself

File: packages/languages/de/README.md (hunk 1 above)
Comment (ChristianLieven) [reply packages/languages/de/README.md:22]: Woops my bad. Also fixed in the latest commit!

File: packages/languages/da-dk/CHANGELOG.md
Comment (ChristianLieven) [reply packages/languages/da-dk/CHANGELOG.md:1]: Alright! I wasn't sure what to do with it. Cleared in the latest commit!

**Pull request merged**